### ISSUE TITLE
bug: The JWT token has already expired, but it can still pass the verification.

### ISSUE THREAD

**Issue by liminggangRs** [apache/apisix#10891]
### Current Behavior

I set up a consumer and configured the JWT plugin, and then I enabled the JWT plugin in the route. At this point, when the token is incorrect, APISIX can properly intercept the request. However, when the token expires, APISIX still allows the request to pass through successfully.

### Expected Behavior

The request should be refused when the jwt token expires

### Steps to Reproduce

run apisix images(apache/apisix:3.2.0-centos)

### Environment

- APISIX version (run `apisix version`): 3.2.0
- Operating system (run `uname -a`): centos
- OpenResty / Nginx version (run `openresty -V` or `nginx -V`):
- etcd version, if relevant (run `curl http://127.0.0.1:9090/v1/server_info`): 3.4.20
- APISIX Dashboard version, if relevant:
- Plugin runner version, for issues related to plugin runners:
- LuaRocks version, for installation issues (run `luarocks --version`):

**Comment by hanqingwu** [comment]
what 's your plugin jwt-auth config value  ?

**Comment by liminggangRs** [comment]
> what 's your plugin jwt-auth config value ?
```
{
  "username": "my username",
  "plugins": {
    "jwt-auth": {
      "algorithm": "HS512",
      "disable": false,
      "exp": 3600,
      "key": "my key",
      "secret": "my secret"
    }
  }
}
```

**Comment by hanqingwu** [comment]
OK, I will try to reproduce,  when I check to code current, I think maybe it not supported yet .

**Comment by hanqingwu** [comment]
@liminggangRs ， I have test it in apisix v3.6.0  And It works , when jwt token expire , apisix will return 

{"message":"failed to verify jwt"}

**Comment by hanqingwu** [comment]
It use resty.jwt:verify_jwt_obj  to  checkout jwt  token  whether verified

**Comment by liminggangRs** [comment]
@hanqingwu Thank you for your assistance, I found the problem. When I was using the API to generate the JWT token, I manually set a 13-digit timestamp as the expiration time in the payload, which overrides the expiration time auto-generated by the jwt-auth plugin. In APISIX, only 10-digit timestamps are processed. When encountering a 13-digit timestamp, it is considered a very large time, which leads to my token appearing as if it will never expire.
